### ISSUE TITLE
Upgrade Django to 3.x

### ISSUE THREAD

**Issue by kimberscott** [lookit/lookit-api#492]
**TL;DR**: Upgrade Django since LTS for 1.1x ends this month

**Narrative**
To continue to take advantage of bug and security patches, we need to upgrade from Django 1.11.x. 

**Acceptance Criteria**

- [x] Django upgraded to 3.0.x
- [x] Modifications to get all dependencies working with this upgrade as needed
- [x] Tests passing
- [x] Rico sign-off

**Implementation Notes**

Only technically need to upgrade to 2.x to avoid losing LTS - but might as well upgrade to 3.x while we're at it to get access to new features.

Review release notes per final version at https://docs.djangoproject.com/en/3.0/releases/ and work through sequentially

For deprecations specifically: https://docs.djangoproject.com/en/3.0/internals/deprecation/

**Comment by Datamance** [comment]
Done with #521 and #520
